Xamarin.Forms.Core.UnitTests/Xamarin.Forms.Core.UnitTests.csproj

```diff
@@ -274,7 +275,7 @@
   <ItemGroup />
   <Target Name="_CopyNUnitTestAdapterFiles" AfterTargets="Build">
     <ItemGroup>
-      <_NUnitTestAdapterFiles Include="$(NuGetPackageRoot)NUnit3TestAdapter\%(Version)\build\net35\**" Condition="@(PackageReference -> '%(Identity)') == 'NUnit3TestAdapter'" InProject="False" />
```

Comment:
should this have changed?

Reply:
Uh. No. But it's added again on the next line but it just escapes the `>` to a `gt;`. I'm guessing that's an inconsistency between VS versions/VSWin and VSMac? I'll revert to be sure